snews_pt/snews_pt_utils.py

```diff
@@ -519,9 +522,19 @@ def is_snews_format(snews_message):
 
     # Time format check
     try:
-        datetime.fromisoformat(snews_message['neutrino_time'])
+        datetime.fromisoformat(snews_message["neutrino_time"])
+
+        # Accept ":" as divider for fractions of a second but change it to "." for CS.
+        split_time = snews_message["neutrino_time"].split(":")
+
+        if len(split_time) == 4:
+            snews_message["neutrino_time"] = ".".join([":".join([split_time[:-1]]), split_time[-1]])
+
+            # Make sure it was the known issue ":" being used for fractions of a second.
+            datetime.fromisoformat(snews_message["neutrino_time"])
+
```

Comment:
Isn't 
```python
dateobj = datetime.fromisoformat(snews_message["neutrino_time"])
datestr = dateobj.isoformat()
snews_message['neutrino_time'] = datestr
```
safer and simpler? This would take anything that can be isoformatted, make a datetime object, and convert that object to string using the default formatting. I can test if this works as intended